wallets/metamask/src/fixture-actions/importAndConnectForFixtures.ts

```diff
@@ -0,0 +1,39 @@
+import type { Page } from '@playwright/test'
+import { MetaMask } from '..'
+import { retryIfMetaMaskCrashAfterUnlock } from '..'
+import { closePopover } from '../pages/HomePage/actions'
+
+export async function importAndConnectForFixtures(
```

Comment:
do you think it may be worth to separate import and connect responsibilities?

Reply:
This replicates the cached ```connected.wallet.setup.ts``` as of now we don't use either of these elsewhere. If we need one or the other later we can separate them.